package.json

```diff
@@ -51,9 +51,10 @@
     "lint:eslint:fix": "npm-run-all \"lint:eslint --fix\"",
     "lint:fix": "npm-run-all lint:*:fix",
     "test": "node bin/run-tests.js",
-    "test:blueprints:js": "EMBER_TYPESCRIPT_BLUEPRINTS=false pnpm test:blueprints:ts",
-    "test:blueprints:ts": "mocha node-tests/blueprints/**/*-test.js",
-    "test:blueprints": "pnpm test:blueprints:js && pnpm test:blueprints:ts",
+    "test:blueprints:js": "EMBER_TYPESCRIPT_BLUEPRINTS=false pnpm test:blueprints:js-from-ts",
+    "test:blueprints:js-from-ts": "mocha node-tests/blueprints/**/*-test.js",
+    "test:blueprints:ts": "mocha node-tests/blueprints-ts/**/*-test.js",
+    "test:blueprints": "npm-run-all test:blueprints:*",
```

Comment:
We can remove this one now since we no longer use it in CI, and I don't expect anyone to run all of them like this locally either.